src/components/ItemDetail/Cite.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import MultiMetadata from './MultiMetadata';
+import SingleMetadata from './SingleMetadata';
+
+const Cite = props => {
+  if (!props.item) {
+    return [];
+  }
+
+  const {
+    title = null,
+    permalink = null,
+    identifier = null,
+    license = null,
+    use_statement = null
+  } = props.item;
+
+  let formatMLA = () => {
+    return [title.primary + " here's the rest MLA"];
+  };
+
+  let formatChicago = () => {
+    return [title.primary + ' chicago format'];
+  };
+
+  let formatAPA = () => {
+    return [title.primary + ' apa format'];
+  };
+
+  let formatWikipedia = () => {
+    return [title.primary + ' wikipedia format'];
+  };
+
+  return (
+    <div>
+      <div className="cite-group-col">
+        <div className="cite-group">
+          <SingleMetadata title="Ark" items={[permalink]} />
```

Comment:
I'd make `item` prop singular and send in a regular value rather than an `array`.